stdlib/dune

```diff
@@ -29,7 +29,11 @@
           (run sed -i s/loc_FUNCTION/loc_POS/ %{input-file})
           (run awk -v dune_wrapped=true
                  -f %{dep:expand_module_aliases.awk} %{input-file})))
-      stdlib))))
+      stdlib)
+     (; FIXME: remove after 4.12 (this erases injectivity annotations)
+      (action (run sed "s/\\!\\([+]*'\\)/\\1/g" %{input-file}))
```

Comment:
maybe `[+-]`?

Reply:
These aren't present yet, just like `+!'a` isn't used (though it apparently could be, if I believe the manual), and this regex is already ugly enough.

This temporary hack isn't robust, thankfully it truly is only temporary.

Reply:
Since I had to rebase anyway, I changed the regexp as you suggested.